src/lib/cli/command.js

```diff
@@ -357,26 +355,28 @@ args.argv = async function( argv, cb ): Promise<any> {
 		}
 
 		switch ( _opts.module ) {
-			case 'import-sql':
+			case 'import-sql': {
 				const site = options.env;
 				if ( site && site.primaryDomain ) {
 					const primaryDomainName = site.primaryDomain.name;
 					info.push( { key: 'Primary Domain Name', value: primaryDomainName } );
 				}
 
 				// Site launched details
-				const haveLaunchedField = site.hasOwnProperty( 'launched' );
+				const haveLaunchedField = Object.prototype.hasOwnProperty.call( site, 'launched' );
```

Comment:
ESLint [recommends this](https://eslint.org/docs/latest/rules/no-prototype-builtins) as a safe way to call `hasOwnProperty`.